src/main/java/Problem2.java

```diff
@@ -7,6 +7,22 @@
     // Do not change signature (function name, parameters and return type)
     // Do not use extra space for another array. Solution to be in-pace with O(1) extra memory
     public static int remove(int[] nums) {
-        return Integer.MAX_VALUE; // place holder
+        int number;
+        int prevNumber;
+        int difNumIndex = 1;
+        for (int i = 1; i < nums.length; i++) {
+            number = nums[i];
+            prevNumber = nums[i-1];
+
+            if (number != prevNumber) {
+                nums[difNumIndex] = number;
+                difNumIndex++;
+            }
+        }
+        for (int i = difNumIndex; i < nums.length; i++) {
```

Comment:
Wrong algorithm. Only one loop is used in O(1) complexity

Reply:
this loop is not needed.